daemon/graphdriver/aufs/aufs.go

```diff
@@ -67,7 +72,7 @@ func init() {
 type Driver struct {
 	root       string
 	sync.Mutex // Protects concurrent modification to active
-	active     map[string]int
+	active     map[string]*data
```

Comment:
I could be wrong but from what I saw this could be just `map[string]data`, no need for pointer.